modules/031-local-path-provisioner/docs/FAQ.md

```diff
@@ -27,3 +27,13 @@ prometheus: |
 ```
 
 Wait for the restart of Prometheus Pods.
+
+## Ho to change retention policy?
+
+At moment delete retention policy is hardcoded, and there is no way to change it [issue](https://github.com/deckhouse/deckhouse/issues/360)
```

Comment:
Retention policy `Delete` is hardcoded, and there is no way to change it [issue](https://github.com/deckhouse/deckhouse/issues/360).